<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(209, 169, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="visor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mostrar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 244, 178);</string>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 205, 253);</string>
    </property>
    <item>
     <property name="text">
      <string>RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Binario</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="Barrita">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="umbral">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 244, 178);</string>
    </property>
    <property name="text">
     <string>Número</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 205, 253);</string>
    </property>
    <property name="text">
     <string>Filtro de mediana</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 205, 253);</string>
    </property>
    <property name="text">
     <string>Filtro de media</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deshacer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 244, 178);</string>
    </property>
    <property name="text">
     <string>Deshacer filtro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aplicar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 244, 178);</string>
    </property>
    <property name="text">
     <string>Aplicar filtro</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>194</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>194</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>¡Fecha de hoy!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 8pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>La puedes cambiar si quieres :)</string>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="visor_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>umbral</zorder>
   <zorder>dateTimeEdit</zorder>
   <zorder>visor</zorder>
   <zorder>comboBox</zorder>
   <zorder>mostrar</zorder>
   <zorder>radioButton2</zorder>
   <zorder>radioButton1</zorder>
   <zorder>aplicar</zorder>
   <zorder>deshacer</zorder>
   <zorder>Barrita</zorder>
   <zorder>label</zorder>
   <zorder>visor_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>Abrir</string>
    </property>
    <addaction name="abrirfoto"/>
    <addaction name="actionCerrar"/>
   </widget>
   <addaction name="menuAbrir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFoto">
   <property name="text">
    <string>Foto</string>
   </property>
  </action>
  <action name="actionDOS">
   <property name="text">
    <string>DOS</string>
   </property>
  </action>
  <action name="abrirfoto">
   <property name="icon">
    <iconset>
     <normaloff>:/iconoparabrir.png</normaloff>:/iconoparabrir.png</iconset>
   </property>
   <property name="text">
    <string>Foto</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
  </action>
  <action name="cerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="cerrar_2">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
